home/web/client/components/AccountsChooser/AccountsChooser.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import DropDownMenu from 'material-ui/DropDownMenu';
 import MenuItem from 'material-ui/MenuItem';
 
-import isEqual from 'lodash.isequal';
```

Comment:
`import { isEqual } from 'lodash'`
so only what's used is included in builds

this way it's much easier to manage